cbmex/cbmex.cpp

```diff
@@ -1763,22 +1763,31 @@ void OnFileConfig(
     // Do a quick look at the options just to find the instance if specified
     for (int i = 1; i < nrhs; ++i)
     {
+        
+        /* Cheat-fix. We know that prhs[ 3 ] is the 'action', by definition
+           of 'fileconfig'. However, a numeric argument will trigger an
+           error inside this for loop. To avoid this, skip input 3. There
+           is code further down that does basic type and scalar checking.
+        */
+        if  ( i == 3 )
+          continue ;
+        
         if (param == PARAM_NONE)
         {
-            char cmdstr[128];
-            if (mxGetString(prhs[i], cmdstr, 16))
+            char cmdstr[255];
+            if (mxGetString(prhs[i], cmdstr, 255))
             {
                 char errstr[128];
                 sprintf(errstr, "Parameter %d is invalid", i);
-                PrintHelp(CBMEX_FUNCTION_COMMENT, true, errstr);
+                PrintHelp(CBMEX_FUNCTION_FILECONFIG, true, errstr);
             }
             if (_strcmpi(cmdstr, "instance") == 0)
             {
                 param = PARAM_INSTANCE;
             }

```

Comment:
add this block:

            else if (_strcmpi(cmdstr, "option") == 0)
            {
                param = PARAM_OPTION;
            }
            else
            {
                param = PARAM_NONE; // Just something valid but instance
            }